README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
 For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
 
 If your project is interested in joining the Node.js Foundation please read the [Project Lifecycle.md](./Project-Lifecycle.md) documentation.
 
+## Projects, Working Groups and the Incubator
+
+The Node.js Foundation is setup to be the home of multiple autonomous projects. Projects are 
+admitted into the incubator and are mentored until their governance and contributor 
+base reach a state of maturity. They are then given a proper charter and a seat on the TSC.
+
+The TSC also forms Working Groups to tackle tasks that don't scope to a single project. These 
+Working Groups can be given a seat on the TSC at the TSC's discretion.
+
+The goal of the Incubator is to:
+
+* Support the Node.js Ecosystem.
+* Provide a neutral "home" for projects so that they are not owned by a single individual or company.
+* Mentor projects in the Node.js community on open governance and liberal contribution policies.
+* Ensure that the Node.js Foundation does not charter projects that do not adhere to the values 
+of the Node.js Foundation. Projects must be: Transparent, Participatory and Effective.
+* The TSC does not "pick winners" and may admit multiple projects that serve similar use cases. 
+Admitting a project for incubation or even chartering it is not an endorsement by the Node.js 
+Foundation or TSC on the technical merits of a project, it is only an endorsement that the project 
+adheres to the TSC's values.
+
+Other than being written in Node.js there are no technical requirements on projects to be admitted
```

Comment:
As already suggested, being written in Node.js should not be a requirement, so "Other than being written in Node.js" should be removed.